inc/aioseop_updates_class.php

```diff
@@ -120,7 +120,13 @@ function do_version_updates( $old_version ) {
 		) {
 			$this->bad_bots_remove_semrush_201810();
 		}
-
+		
+		if (
+			( ! AIOSEOPPRO && version_compare( $old_version, '3.0', '<' ) ) ||
```

Comment:
Since we're merging the version numbers now we don't need to distinguish between free and pro when it comes to version numbers.